apps/epicdev-ai/src/lib/google-calendar.ts

```
@@ -0,0 +1,133 @@
+				`Google Calendar API Error: ${error.response.data.error.message || 'Unknown API error'} (Code: ${error.response.data.error.code})`,
+			)
+		} else {
+			console.error('Full Error:', error.message || error)
+			throw new Error(
+				`Failed to create calendar event: ${error.message || 'Unknown error'}`,
+			)
+		}
+		// Note: Removed the specific troubleshooting suggestions here as they are less relevant for a general library function error.
+		// The calling code should handle the thrown error.
+		console.error('------------------------------------------------------')
+		throw error
```

Biome (1.9.4)

[error] This code is unreachable

... because either this statement, ...

... this statement, ...

... this statement, ...

... this statement, ...

... this statement, ...

... or this statement will throw an exception beforehand


(lint/correctness/noUnreachable)